ptsprojects/zephyr/gatt_wid.py

```diff
@@ -41,11 +41,16 @@ def gatt_wid_hdl(wid, description, test_case_name):
 
 def gatt_server_fetch_db():
     db = GattDB()
+    char_val_set = set()
```

Comment:
In fact it was intended. I don't like this patch, because GattDB will lack Characteristic Values. Please add new class representation for those attributes